torchvision/datapoints/__init__.py

```diff
@@ -12,3 +12,25 @@
     import warnings
 
     warnings.warn(_BETA_TRANSFORMS_WARNING)
+
+
+def wrap(wrappee, *, like, **kwargs):
```

Comment:
```suggestion
D = TypeVar("D", bounds=Datapoint)

def wrap(wrappee: torch.Tensor, *, like: D, **kwargs: Any) -> D:
```

Reply:
I had merged the suggestion and fixed the imports but it created more problems in the BBox._wrap, so I'm getting rid of it. Mypy is green anyway.

Reply:
I'll have a look.